.github/scripts/createReleaseNotes.js

```diff
@@ -0,0 +1,132 @@
+// createReleaseNotes.js
+const { appendSummary, getReleaseVersionValue } = require("./actionUtils");
+
+/**
+ * Formats the current date in a specific string format for use in release titles.
+ *
+ * @returns {string} The formatted date string in the format: DD MMM YYYY (e.g., "15 MAY 2024").
+ */
+function formatDate() {
+  const date = new Date();
+  const options = { day: "2-digit", month: "short", year: "numeric" };
+  return date.toLocaleDateString("en-US", options).toUpperCase();
+}
+
+/**
+ * Creates a draft release on GitHub with a specified tag and release notes.
+ *
+ * @param {Object} github The authenticated GitHub API client instance.
+ * @param {string} owner The username or organization name on GitHub that owns the repository.
+ * @param {string} repo The repository name.
+ * @param {string} tag_name The tag associated with the release.
+ * @param {string} body The content of the release notes.
+ * @returns {Promise<string>} A Promise that resolves with the URL of the newly created draft release.
+ */
+async function createDraftRelease(github, owner, repo, tag_name, body) {
+  try {
+    const response = await github.rest.repos.createRelease({
+      owner,
+      repo,
+      tag_name,
+      name: `${tag_name} - ${formatDate()}`,
+      body,
+      draft: true,
+      prerelease: false,
+    });
+
+    const releaseUrl = response.data.html_url; // Extract URL from the response object
+    console.log("Release URL:", releaseUrl); // Log URL to the console
+    // console.log('Release created successfully:', response);
+    return releaseUrl; // Return the URL
```

Comment:
These are not helpful comments.
```suggestion
    const releaseUrl = response.data.html_url;
    console.log("Release URL:", releaseUrl);
    return releaseUrl;
```

Reply:
thank you